zazu/util.py

```diff
@@ -95,20 +95,30 @@ def prompt(text, default=None, expected_type=str):
     return expected_type(result)
 
 
-def pick(choices, message):
+def pick(choices, message, checkbox=False):
+
     if len(choices) > 1:
```

Comment:
Well its either they want it or not right ? We collect the options from the registered plugins, check box just allows a choice to include in the yaml config